verifiers/internal/gha/npm.go

```diff
@@ -21,6 +21,8 @@ import (
 
 const (
 	publishAttestationV01 = "https://github.com/npm/attestation/tree/main/specs/publish/"
+	builderGitHubRunnerID = "https://github.com/actions/runner"
+	ossfNpmBuilderID      = "https://github.com/slsa-framework/slsa-github-generator/.github/workflows/builder_nodejs_slsa3.yml"
```

Comment:
I guess you don't actually need this anymore since the builder-id option is required?

Reply:
Good catch. You're correct, I'll remove it.